src/plugins/spec/spec-new.c

```diff
@@ -0,0 +1,225 @@
+#include "spec-new.h"
+#include "kdberrors.h"
+#include "kdbglobbing.h"
+
+#include <kdbhelper.h>
+
+#ifdef __MINGW32__
+static bool specMatches (Key * specKey, Key * otherKey)
+{
+	/**
+	 * Known limitation: For MINGW builds fnmatch.h does not exist. Therefore, globbing can't be used.
+	 * This means that there is no support for # and _ in key names.
+	 * This function was copied from 68e9dff, doesn't use globbing and therefore doesn't require the globbing library which is not
+	 * compatible with Windows:
+	 */
+	const char * spec = keyUnescapedName (specKey);
+	size_t specNsLen = strlen (spec) + 1;
+	spec += specNsLen; // skip namespace
+	const char * other = keyUnescapedName (otherKey);
+	size_t otherNsLen = strlen (other) + 1;
+	other += otherNsLen; // skip namespace
+	size_t const specSize = keyGetUnescapedNameSize (specKey) - specNsLen;
+	size_t const otherSize = keyGetUnescapedNameSize (otherKey) - otherNsLen;
+
+	return specSize == otherSize && memcmp (spec, other, specSize) == 0;
+}
+#else
+/**
+ * Check whether the {@link otherKey} matches the {@link specKey}.
+ *
+ * @param specKey specification key
+ * @param otherKey the other key to match the specification key with
+ * @return 1 - if the keys match
+ * 	   0 - if the keys do not match
+ */
+static bool specMatches (Key * specKey, Key * otherKey)
+{
+	// ignore namespaces for globbing
+	Key * globKey = keyNew (strchr (keyName (otherKey), '/'), KEY_END);
```

Comment:
See my comment about `keyName` below. It will contain the `otherKey`'s namespace.

Reply:
You want to use `strchr (keyName (otherKey), '/')`

Reply:
Already using it here, or do you mean something else?